activerecord/lib/active_record/log_subscriber.rb

```diff
@@ -132,6 +137,10 @@ def log_query_source
       def extract_query_source_location(locations)
         backtrace_cleaner.clean(locations.lazy).first
       end
+
+      def parameter_filters
+        ActiveSupport::ParameterFilter.new(filter_parameters)
```

Comment:
We can change this implementation to `ActiveRecord::Base.inspection_filter` and remove the need to define `filter_parameters` in the log subscriber class.

That also allows us to change the test to remove the stub for this method since we can just configure the parameter filter attributes in `ActiveRecord::Base` and return its original value after the test.

Reply:
Nice suggestion. 

Updated the PR to use filter attributes method in `ActiveRecord::Base` and updated the test accordingly.